time.js

```diff
@@ -8,21 +8,318 @@
 require('@js-joda/timezone');
 const time = require('@js-joda/core');
 
-// openHAB uses a RFC DateTime string, js-joda defaults to the ISO version, this defaults RFC instead
+const javaZDT = Java.type('java.time.ZonedDateTime');
+const javaDuration = Java.type('java.time.Duration');
+const javaString = Java.type('java.lang.String');
+const javaNumber = Java.type('java.lang.Number');
+const { DateTimeType, DecimalType, StringType, QuantityType } = require('@runtime');
+const ohItem = Java.type('org.openhab.core.items.Item');
+
+/**
+ * Converts the Java ZonedDateTime to a time.ZonedDateTime
+ * @param {java.time.ZonedDateTime} zdt date time to convert to a time.ZonedDateTime
+ * @returns {time.ZonedDateTime}
+ */
+const javaZDTtoZDT = function(zdt) {
+    const epoch = zdt.toInstant().toEpochMilli();
+    const instant = time.Instant.ofEpochMilli(epoch);
+    const zone = time.ZoneId.of(zdt.getZone().toString());
+    return time.ZonedDateTime.ofInstant(instant, zone);
+}
+
+/**
+ * Adds millis to the passed in ZDT as milliseconds. The millis is rounded 
+ * first. If millis is negative they will be subtracted.
+ * @param {number|bigint} millis number of milliseconds to add
+ */
```

Comment:
The `@private` makes sense to me but why the `@memberof time`? It's not part of `time` like, for example `time.toZDT()` is. Is it because they are called from `time.toZDT()`?

Reply:
Oh sorry, I just checked, it is not exported.
You are totally right, `@memberof time` makes no sense here. I am not sure why I wrote that yesterday, must have been a misthought.